### ISSUE TITLE
Seafile Integration OnlyOffice Open File Exception

### ISSUE THREAD

**Issue by cfy3133** [haiwen/seafile#2763]
Hello author, after integrating seafile with OnlyOffice, if the open file size is greater than 20MB, an error conversion timeout will be reported. If the open file size is less than 20MB, it can be opened normally, and opening files larger than 20MB using OnlyOffice alone is also normal
![image](https://github.com/haiwen/seafile/assets/43978367/d4b55a20-3f5b-49d7-a765-0976c82f8baf)
![image](https://github.com/haiwen/seafile/assets/43978367/d9d5ddbe-4685-4e81-a7f3-9b2e4d47b702)
Seafile is the latest version
![image](https://github.com/haiwen/seafile/assets/43978367/3dd96cd1-2638-428b-aeda-ee432577e25b)

**Comment by hg2s** [comment]
I am seeing a similar error when opening large files via the web interface. Is there a configuration setting for maximum file size in OnlyOffice? I am not aware of any setting for this. 

![onlyoffice-error](https://github.com/haiwen/seafile/assets/54009095/b3366f98-0436-4788-853f-c39ec518c16a)

Seafile CE 11.0.8 | Ubuntu 22.04 LTS | OnlyOffice 8.0.1.31

**Comment by VPaulV** [comment]
I don't think it is a seafile issue - https://forum.onlyoffice.com/t/file-exceeds-the-limitation-set-for-your-server/7504
